ldap/servers/slapd/back-ldbm/cache.c

```diff
@@ -638,6 +643,11 @@ flush_hash(struct cache *cache, struct timespec *start_time, int32_t type)
     }
 
     cache_unlock(cache);
+
+    clock_gettime(CLOCK_MONOTONIC, &flush_end);
+    slapi_timespec_diff(&flush_end, &flush_start, &duration);
+    snprintf(flush_etime, ETIME_BUFSIZ, "%" PRId64 ".%.09" PRId64 "", (int64_t)duration.tv_sec, (int64_t)duration.tv_nsec);
+    slapi_log_err(SLAPI_LOG_WARNING, "flush_hash", "Upon BETXN callback failure, entry cache is flushed during %s\n", flush_etime);
```

Comment:
I do not know how frequent this message is displayed in a real deployment 
 and I wonder if WARNING is not a bit too frightening (Maybe INFO would be better )

Reply:
I had the same concern. Failure of a plugin is not a big deal but revert_cache is so slow (especially with large entry cache) and almost freeze the server (all requests accessing to the entry cache are stuck) that I was thinking the first thing that an admin would do is to check the error log. Making it a warning may match what LDAP clients reports.